contrib/graphite/scripts/bootstrap.sh

```diff
@@ -95,7 +95,7 @@ ln -s /etc/nginx/sites-available/grafana /etc/nginx/sites-enabled/grafana
 /etc/init.d/nginx restart
```

Comment:
I have no idea either, I just seach for and replace the reference.  My guess is that this is an old version of dockerfile for blueflood... i think this is how the original docker setup got started maybe?  Similar to how graphite-api and finder was started?  We can probably delete this out of blueflood/contrib eventually (along with the contrib/graphite).